test/SpecHelper.js

```diff
@@ -32,7 +32,7 @@ chai.use(function(chai, utils) {
 	Assertion.addMethod('closeToLatLng', function(actual, delta, message) {
 		var obj = utils.flag(this, 'object');
 
-		delta = delta || 1e-4;	
+		delta = delta || 21e-2;	
```

Comment:
Can you please explain why we had to change this value and how you chose the new value? Didn't look very deeply into this myself yet so sorry if this is totally off but is it related to this merged PR in a later version of Leaflet by any chance? https://github.com/Leaflet/Leaflet/pull/5935 / would this therefore be something that would be fixed by an upgrade rather than by expanding our own testing definition?